spec/acceptance/03_no_ssl_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper_acceptance'
+
+describe 'sensu without SSL', unless: RSpec.configuration.sensu_cluster do
```

Comment:
@ghoneycutt This is the tests where `use_ssl => false` is done using method documented in README.